servers/src/main/java/tachyon/master/lineage/meta/LineageStore.java

```diff
@@ -110,15 +109,6 @@ public synchronized void deleteLineage(long lineageId) throws LineageDoesNotExis
     LineageDoesNotExistException.check(mIdIndex.containsKey(lineageId),
         ExceptionMessage.LINEAGE_DOES_NOT_EXIST, lineageId);
 
-    deleteLineage(lineageId, Sets.<Long>newHashSet());
-  }
-
-  private void deleteLineage(long lineageId, Set<Long> deleted)
```

Comment:
This set is not used in this code, and it should be unnecessary assuming it's a DAG.